lib/riemann.js

```diff
@@ -49,7 +49,7 @@ Riemann.prototype.onPacket = function(packet, rinfo) {
 };
 
 Riemann.prototype.parsePacket = function(packet) {
-  return packet.toString().split('\n');
+  return packet.toString().split('\n').filter(function(el) {return el.length != 0});
```

Comment:
The `trim` solution is more efficient (about twice as fast over 1000000 runs), and a bit clearer too.